src/main/kotlin/day14/practice problem 01.kt

```
@@ -10,7 +10,11 @@ data class Todo(
     val id: Int,
     val title: String,
     val completed: Boolean,
-)
+) {
+    companion object {
+
+    }
```

detekt (1.23.7)

[warning] The class or object Companion is empty.

(detekt.empty-blocks.EmptyClassBlock)